syntax.php

```diff
@@ -72,11 +72,11 @@ function _link($target) {
         static $url = '^https?://';
         // '^(http://)?[-_[:alnum:]]+[-_.[:alnum:]]*\.[a-z]{2}'
         // '(/[-_./[:alnum:]&%?=#]*)?';
-        if (ereg($url, $target))
+        if (preg_match($url, $target))
             return $target;
 
         /* Match an internal link. */
-        list($id, $hash) = split('#', $target, 2);
+        list($id, $hash) = preg_split('#', $target, 2);
```

Comment:
this split should be changed into an explode

Reply:
It works with delimiters:

`       list($id, $hash) = preg_split('/#/', $target, 2);`

or with explode:

`       list($id, $hash) = explode('#', $target, 2);`